### DISCUSSION TITLE
When I run this code, it throw an error as below, how can i deal with it

### DISCUSSION THREAD

**Discussion by PURESUN-OYLY** [marksgraham/OCT-Converter#60] [General]
Exception has occurred: RuntimeError No decoders are available

Full logs:
![image](https://user-images.githubusercontent.com/52588971/181274921-5fe9451e-904b-4326-9e8b-ad6dd517e041.png)

**Comment by PURESUN-OYLY** [comment]
I TRY TO DECODE FDA FILE

**Comment by marksgraham** [comment]
Are you running on windows? I've seen the error before, see here: https://github.com/marksgraham/OCT-Converter/issues/52

Seems to be to do with your installation of pylibjpeg, can you try installing in a fresh environment?
